.github/workflows/beta.yml

```diff
@@ -0,0 +1,55 @@
+name: Beta branch CI
```

Comment:
What is the upside of building out a second workflow, as opposed to having one workflow with a matrix that includes stable, beta and dev?